pi1/class.tx_css2inline_pi1.php

```diff
@@ -53,11 +53,13 @@ class tx_css2inline_pi1 extends \TYPO3\CMS\Frontend\Plugin\AbstractPlugin
      */
     public function main($content, array $conf)
     {
+        /*
         // Require 3rd-party libraries, in case TYPO3 does not run in composer mode
         $pharFileName = \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::extPath($this->extKey) . 'Libraries/pelago-emogrifier.phar';
         if (is_file($pharFileName)) {
             @include 'phar://' . $pharFileName . '/vendor/autoload.php';
         }
+        */
```

Comment:
with that change, the extension can unfortunately not be installed through TER and this is uncool even if I'm the first one to only use composer nowadays :(

Reply:
That's right. And I'm well aware that you spent some time for that code and that in composer.json.

Reply:
is it possible to make a branch for a composer-only-version "typo3_9_composer" or somehow named like that?
Like that at least it's available for composer.
I don't know if it's very problematic to solve the problem with the phar-files outside the extension-directory, but storing it in the extension-directory is probably not so clean as solution.